app/scripts/controllers/necesidad/solicitud_necesidad.js

```diff
@@ -82,11 +80,7 @@ angular.module('contractualClienteApp')
         self.FormularioSeleccionado = 0;
         self.tipoInterventor = false;
 
-        // para mostrar select de plan adquisicion, no existe servicio
-        self.planes_anuales = [{
-            Id: 1,
-            Nombre: "Plan de Adquisición 2019"
-        }];
+        self.planes_anuales = [{}];
```

Comment:
- `[{}]`: Un arreglo de longitud 1, cuyo único elemento es un objeto vacío
- `[]`: Un arreglo vacío

```suggestion
        self.planes_anuales = [];
```

Reply:
Se modifica para dejarlo como un arreglo vacío

Reply:
Se modifica para dejarlo como un arreglo vacío